systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -110,16 +111,17 @@
     static final long TEARDOWN_GLOBAL_WAIT = 10000;
 
     public static KubeClusterResource cluster = new KubeClusterResource();
-    protected static DefaultKubernetesClient client = new DefaultKubernetesClient();
+    protected static DefaultOpenShiftClient client = new DefaultOpenShiftClient();
```

Comment:
Why was this needed?

Reply:
I think it's not necessary now. I did some experiments with RoleBindings, because I wasn't successful with deployment. I will put there `DefaultKubernetesClient` back.